lib/DhallToCabal/ConfigTree.hs

```diff
@@ -33,7 +33,7 @@ instance ( Semigroup a ) => Semigroup ( ConfigTree cond a ) where
 
 instance ( Monoid a ) => Monoid ( ConfigTree cond a ) where
   mempty = pure mempty
-  mappend = liftM2 mappend
+  mappend = (<>)
```

Comment:
This might be breaking GHC 7.10 compatibility; the warning is unfortunate but it's the least evil. I don't actually care too much about old GHCs, but bounds will need to be adjusted and I think @jneira does care about GHC 7.10.

Reply:
No problem, i have already to keep my etlas specific fork with changes over ghc and cabal versions so updating the bounds could be fine